KelynPNjeri/Store-Manager
# #161041511 View Quantity of a Product in Inventory

#### What does this PR do?
This PR allows a Store Owner to view the quantity of a product in inventory.

#### Description of Task to be completed?
The task to be completed was to create a means by which a Store Owner could view the amount of a product in the inventory.

#### How should this be manually tested?
You can clone into my repository using the git clone `git clone https://github.com/KelynPNjeri/Store-Manager.git` to your desired location.
Go to the location to which you cloned the repository.
Change directory into the Store Manager directory using `cd Store-Manager`
Navigate to the ft-view-quantity-161041511 branch using the `git checkout ft-view-quantity-161041511` command.
Launch the `product.html` file in the html folder using your favorite web browser.
Click on any of the product's images to view details of a product and among those details, you'll see the quantity of a product.

#### Any background context you want to provide?
None.

#### What are the relevant pivotal tracker stories?
[#161041511](https://www.pivotaltracker.com/story/show/161041511)

#### Screenshots (if appropriate)
![quantity](https://user-images.githubusercontent.com/34427048/46829846-a0742980-cda7-11e8-881a-92bb4199f0dd.png)
